docs/java-custom-catalog.md

```diff
@@ -30,7 +30,7 @@ menu:
 It's possible to read an iceberg table either from an hdfs path or from a hive table. It's also possible to use a custom metastore in place of hive. The steps to do that are as follows.
 
 - [Custom TableOperations](#custom-table-operations-implementation)
-- [Custom Catalog](#custom-catalog-implementation)
+- [Custom Catalog](#custom-catalog-implementation-1)
```

Comment:
I can confirm that this link is broken, but any idea why it requires a `-1`?

Reply:
@nastra: Because the heading name is also  `Custom Catalog Implementation` (#) and a subsection name is also `Custom Catalog Implementation` (###)
Hence. anchor tags add a `-1` suffix to generate a unique hyperlink

Reply:
I am even ok to rename the heading as "Custom catalog in Iceberg" or just "Custom catalog"

Reply:
+1 ro renaming the sub-section so that we don't need the `-1` in the link

Reply:
Sounds good.
Would you want me to change `Custom Catalog Implementation` (#) to `Custom Catalog` and revert the current modification?

Reply:
yes, let's rename `### Custom catalog implementation` to `### Custom catalog` and update `[Custom Catalog](#custom-catalog-implementation)` to `[Custom Catalog](#custom-catalog)`

Reply:
Do you think that maybe the heading of the page should be changed to `# Custom catalog` so that the sub-headings will remain consistent with the `implementation` suffix?

Reply:
that would also work

Reply:
done